MojioSDK/Extensions/CompoundWordStyle.swift

```diff
@@ -26,6 +26,13 @@ public extension RawRepresentable where Self: CompoundWordStyle, RawValue == Str
     }
     
     var lowerCamelCase: String {
-        return self.rawValue.decapitalizedFirstCharacter
+        let value = self.rawValue
```

Comment:
Maybe better to use only lowerCased to compare strings instead of lowerCamelCase?